BLOG: Team delivery improvements (#231)
2i2c-org/2i2c-brochure: content/blog/2024/delivery-improvements/index.md

Across each level of the organisation, we have been experimenting more deliberately with Agile delivery practices. These experiments shape our approach to planning (at both the team and organization-level), visualizing and managing our work streams, and improving our internal product delivery processes. We are agnostic to a specific Agile framework, and have found the most value in blending elements of [Kanban](https://asana.com/resources/what-is-kanban), [Scrum](https://www.agilealliance.org/glossary/scrum/) and [Flight levels](https://flightlevels.io). We will continue to experiment with these practices, seeking to find a happy medium for 2i2c. We'll share more on this soon :)

**Coaching**.
Some members of our team have also begun benefiting from having regular 1:1 coaching to help their interpersonal development. Our team's coaching support includes work to empower improving self-awareness, overcoming self-limiting challenges and gaining technical leadership mastery.
Coupled with our growing practices around team feedback, we aim to build a culture where team members have both internal and external support to help them grow.

{{% callout %}}

> This work was done in partnership with [April Johnson, a Transformation Consultant and Coach](https://www.linkedin.com/in/apriljohnson/).

Author: ```suggestion
This work was done in partnership with [April Johnson, a Transformation Consultant and Coach](http://www.apriljohnson.io/).
```

---
title: Improvements to our team's planning and delivery
date: "2024-04-15"
lastmod: "2024-04-15"
authors: ["Chris Holdgraf", "Harold Campbell"]
tags: [culture]
categories: [updates]
featured: false
draft: false
---

> ---
> 

Author: ```suggestion
```

For example, we'd like to find more efficient ways of coordinating across our team, and encouraging team members to own their own work.

### Changes to our team culture

In addition to our system around planning work, we've also identified a number of ways that we can improve our team's culture related to work.
As a distributed organization, one of our biggest challenges has been the additional friction that comes with communicating across many time zones.
It's much harder to solve a problem or get help when responses come with long gaps of time.

As a result, we've defined several practices that will help our team members grow their autonomy and independence, while still moving towards the same targets and goals.
This is ongoing work that we aim to continue developing over the next few months.

**Personal growth**. 
First, we've started an audit of 2i2c's organizational roles and the responsibilities and the pathways for growth with each of them.
This began with our engineering team, but will continue with other areas of the organization as well.

> We believe that defining personal growth trajectories will help identify the gaps in our team's skills that make it more difficult to act efficiently and independently.

Reviewer: Could we change "personal growth" to "professional growth" or "professional development"? I'd like to highlight that the growth is focused on people's professional skills and qualities.

> **Personal growth**. 
> First, we've started an audit of 2i2c's organizational roles and the responsibilities and the pathways for growth with each of them.
> This began with our engineering team, but will continue with other areas of the organization as well.
> We believe that defining personal growth trajectories will help identify the gaps in our team's skills that make it more difficult to act efficiently and independently.

Author: ```suggestion
**Professional growth**. 
First, we've started an audit of 2i2c's organizational roles and the responsibilities and the pathways for growth with each of them.
This began with our engineering team, but will continue with other areas of the organization as well.
We believe that defining professional growth trajectories will help identify the gaps in our team's skills that make it more difficult to act efficiently and independently.
```

Across each level of the organisation, we have been experimenting more deliberately with Agile delivery practices. These experiments shape our approach to planning (at both the team and organization-level), visualizing and managing our work streams, and improving our internal product delivery processes. We are agnostic to a specific Agile framework, and have found the most value in blending elements of [Kanban](https://asana.com/resources/what-is-kanban), [Scrum](https://www.agilealliance.org/glossary/scrum/) and [Flight levels](https://flightlevels.io). We will continue to experiment with these practices, seeking to find a happy medium for 2i2c. We'll share more on this soon :)

**Coaching**.
Some members of our team have also begun benefiting from having regular 1:1 coaching to help their interpersonal development. Our team's coaching support includes work to empower improving self-awareness, overcoming self-limiting challenges and gaining technical leadership mastery.
Coupled with our growing practices around team feedback, we aim to build a culture where team members have both internal and external support to help them grow.

{{% callout %}}

> This work was done in partnership with [April Johnson, a Transformation Consultant and Coach](https://www.linkedin.com/in/apriljohnson/).

Reviewer: My website is http://www.apriljohnson.io
